_posts/2017-09-30-End-to-end-IoT.md

```diff
@@ -0,0 +1,188 @@
+
+The platform consists of three main parts:
+- MyThings Builder: Charts and sensor values (containers)
+- Mythings Manager: Online device onboarding and user management
+- MyThings Scanner: Offline (in the field) device onboarding
+
+
+<p style="text-align: center;">
+  <img class="image fit" style="width: 250px; margin:0px auto;" alt="NodeJS" src="../img/end-to-end-iot/node.png">
+</p>
+### Node.js &amp; TypeScript
+
+Most people should be familiar with [Node](https://nodejs.org/en/).
+It is the JavaScript runtime built upon the V8 engine that Google Chrome uses.
+It is a lightweigth and efficient runtimle that uses an event-driven, non-blocking I/O model.
+This combined with the added type safety that TypeScript provides makes this an excellent choise for rapid prototyping.
+
+Some of our own demo applications make use of the Node Simple Server (NSS) application, while other just use express. This depends on the needs of the project.
+If you are interested in the NSS project, we have a blogpost about it [here](http://ordina-jworks.github.io/iot/2017/01/21/Node-with-TypeScript.html) and it is [on github](https://github.com/ordina-jworks/NodeSimpleServer) too!
+
+
+<p style="text-align: center;">
+  <img class="image fit" style="height: 200px; margin:0px auto;" alt="Angular" src="../img/end-to-end-iot/angular.png">
+</p>
+### Angular
+
+Like with Node, most people should be familiar with [Angular](https://angular.io) (or the older AngularJS).
+Angular is a development platform for building modern single page web applications.
+It is a complete rewrite of the older AngularJS and therefor has some big changes in how things work.
+Angular is easy to set up and use, it also is fully cross platform/browser compatible.
+
+
+## Our demo applications
+All our demo applications are publicly available in the [github project of NSS](https://github.com/ordina-jworks/NodeSimpleServer).
+These demo application are ever evolving as we are currently porting them from the older AngularJS to Angular with TypeScript.
+
+Our demo applications have been showcased and used at several events including internal Proximus events as well as conferences like Devoxx, Techorama and The Belgian IoT convention in Mechelen.
+
+Below we will go into detail about each application and how it came to be, as well as the iterations they went through.
+
+Aside from the Slotmachine and the Booze-o-meter we've also developed the Stairway to Health application for Proximus. 
+A blogpost about this will be available in October.
+For the impatiant, The IoT talk at the annual JWorks JOIN event convered this topic already and can be viewed on YouTube.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/BbnwrvfozUs?rel=0" frameborder="0" allowfullscreen></iframe>
+
+
+### The slotmachine
+The slotmachine application does mostly what its name suggests, but with a twist.
```

Comment:
Slotmachine is written with a capital in certain places and without capital in other places.
Might be best to use to same notation everywhere.